src/Toolkit/Toolkit.Maui/BookmarksView/BookmarksView.cs

```diff
@@ -66,20 +69,7 @@ public BookmarksView()
     protected override void OnApplyTemplate()
     {
         base.OnApplyTemplate();
-
-        if (_presentingView != null)
-        {
-            _presentingView.ItemSelected -= Internal_bookmarkSelected;
-        }
-
-        _presentingView = GetTemplateChild("PresentingView") as ListView;
-
-        if (_presentingView != null)
-        {
-            _presentingView.ItemSelected += Internal_bookmarkSelected;
-            _presentingView.ItemTemplate = ItemTemplate;
-            _presentingView.ItemsSource = _dataSource;
-        }
```

Comment:
Hooking up and unhooking from events on view elements needs to stay in OnApplyTemplate.
The change also means if a template gets changed, the old listener is still hanging around and

Reply:
There is a call to `UpdatePresentingView` in `ItemTemplateChanged` handler [here](https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/pull/598/files#diff-b515b77dae873f62a7192e60f5ac03fd034dd3bb71c44393ca4cef9adc673752R150).

Reply:
I have corrected this code change.